__tests__/test-utils.js

```diff
@@ -179,6 +181,18 @@ class TestUtils {
   static reorderHandlers() {
     Object.values(httpMock.handlers).forEach(handler => handler.reverse());
   }
+
+  static initLogger() {
+    TestUtils.logger = winston.createLogger({
+      level: 'silly',
+      transports: [
+        new winston.transports.Console({
+          format: winston.format.simple(),
+          level: 'silly',
+        }),
+      ],
+    });
+  }
```

Comment:
This is a simple version of the logger of the integration tests, this was made because jest has some issues with console outputs within the test methods which we usually use for debugging.